index.js

```diff
@@ -261,7 +261,7 @@ exports.extract = function (cwd, opts) {
     var onlink = function () {
       if (win32) return next() // skip links on win for now before it can be tested
       xfs.unlink(name, function () {
-        var srcpath = path.resolve(cwd, header.linkname)
+        var srcpath = path.resolve(cwd, path.join('/', header.linkname))
```

Comment:
This won't work - what you actually need to do is call `var srcpath = path.join(cwd, path.join('/', header.linkname))`